### PR TITLE
concise code

### PR DESCRIPTION
**Pull request by tashrifbillah**

### CODE REVIEW

File: lochness/mediaflux/__init__.py [hunk 1]
```diff
@@ -142,14 +142,14 @@ def sync_module(Lochness: 'lochness.config',
                             subj_dir = subject.protected_folder \
                                 if protect else subject.general_folder
 
-                            mf_local = str(tree.get(datatype,
```
Comment (tashrifbillah) [inline lochness/mediaflux/__init__.py:145]: Get rid of this `str`

File: lochness/mediaflux/__init__.py [hunk 2]
```diff
@@ -142,14 +142,14 @@ def sync_module(Lochness: 'lochness.config',
...
-                                                        subpath.parent)
+                            mf_local = tree.get(datatype,
+                                            subj_dir,
+                                            processed=processed,
+                                            BIDS=Lochness['BIDS']) / \
+                                                    subpath.parent
 
-                            mf_local = str(Path(mf_local) / prod['out_dir']) \
-                                    if 'out_dir' in prod else mf_local
+                            mf_local = str(mf_local / prod['out_dir'] \
```
Comment (tashrifbillah) [inline lochness/mediaflux/__init__.py:151]: * Get rid of `Path` as well
* Apply `str` to final `mf_local`

File: lochness/mediaflux/__init__.py [hunk 3]
```diff
@@ -68,7 +68,7 @@ def sync_module(Lochness: 'lochness.config',
                     - vendor: Activinsights
                       product: GENEActiv
                       data_dir: PrescientXX_Actigraphy
-                      pattern: '*.*'
+                      pattern: '*'
```
Comment (tashrifbillah) [inline lochness/mediaflux/__init__.py:71]: They are the same!

File: lochness/mediaflux/__init__.py (hunk 2 above)
Comment (kcho) [reply lochness/mediaflux/__init__.py:151]: is `mf_local` already Path by this point if we follow https://github.com/AMP-SCZ/lochness/pull/69#discussion_r786300908 ?

Comment (tashrifbillah) [reply lochness/mediaflux/__init__.py:151]: Yes it is.

**Review by kcho — APPROVED** [review]

File: lochness/mediaflux/__init__.py (hunk 2 above)
Comment (tashrifbillah) [reply lochness/mediaflux/__init__.py:151]: You have already answered why:

> which works with /

**Comment by kcho** [comment]
Please merge whenever you're ready

**Pull request merged**